zookeeper-jute/src/main/java/org/apache/jute/ToStringOutputArchive.java

```diff
@@ -90,14 +85,14 @@ public void writeDouble(double d, String tag) throws IOException {
     
     public void writeString(String s, String tag) throws IOException {
         printCommaUnlessFirst();
-        stream.print(Utils.toCSVString(s));
+        stream.print(escapeString(s));
         throwExceptionOnError(tag);
     }
     
-    public void writeBuffer(byte buf[], String tag)
+    public void writeBuffer(byte[] buf, String tag)
     throws IOException {
         printCommaUnlessFirst();
-        stream.print(Utils.toCSVBuffer(buf));
+        stream.print(escapeBuffer(buf));
         throwExceptionOnError(tag);
     }
```

Comment:
- `escapeBuffer` and `escapeString` can still be put into the `Utils.java`?
- conservative and prudent about this PR. Do you have any test evidences about Backward compatibility？

Reply:
1. `escapeBuffer` and `escapeString` originally for `CSVOutputArchive`, in fact we don't need them anywhere outside and thus to be private. As mentioned above, I keep `CSVOutputArchive`(rename to `ToStringOutputArchive`) for backward compatibility(`toString` method of jute generated classes).

2. As linked in description(http://zookeeper-user.578899.n2.nabble.com/Deprecated-CSVInputArchive-and-XMLInputArchive-td7584086.html), we actually remove support for XML/CSV RecFormat, and thus there is no backward compatibility guarantee.